python/src/FunctionalChaosRandomVector_doc.i.in

```diff
@@ -23,3 +23,176 @@ Returns
 -------
 functionalChaosResult : :class:`~openturns.FunctionalChaosResult`
     The result from a functional chaos decomposition."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosRandomVector::getMean
+"Accessor to the mean of the functional chaos expansion.
+
+We consider the following functional chaos expansion:
+
+.. math::
+
+      \widetilde{\vect{Y}} = \sum_{k = 0}^P \vect{a}_k \psi_k(\vect{Z})
+
+where
+:math:`n_Y \in \Nset` is the dimension of the output random vector,
```

Comment:
This is a little clumsy, n_Y does not appear in the equation so its definition should not be after "where"

Reply:
Fixed at fe4d5582158cddc3761e1bb14fbd544b19ed67b2